docs/articles/documentation/reference/test-api/testcontroller/expect/contains.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+title: t.expect.contains Method
+permalink: /documentation/reference/test-api/testcontroller/expect/contains.html
+---
+# t.expect.contains Method
+
+Asserts that `actual` contains `expected`.
+
+```text
+await t.expect( actual ).contains( expected, message, options );
+```
+
+Parameter              | Type                                              | Description
+---------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------
+`actual`             | String &#124; Array &#124; Object | A value we expect to contain the `expected`. See [`actual` parameter value](#actual-parameter-value).
```

Comment:
> A value we expect to contain

I don't think the second person is good here. It is fine in blogs, I could even use it in guides (though the correctors wouldn't approve it), but not in the reference definitely.

Can be something like:

``A value tested in the assertion. The assertion passes if the `actual` value contains `expected`.``